### PR TITLE
Drop support for ASP.NET Core 2.x

### PR DESCRIPTION
**Pull request by andriizhegurov-okta**
## Description:
- **What's changed?** 
ASP.Net Core 2.x code samples were removed from the guides because it's no longer supported
- **Is this PR related to a Monolith release?** 
Not related
### Resolves:

OKTA-330885 ASP.NET > Drop support for ASP.NET Core 2.x

### CODE REVIEW

File: packages/@okta/vuepress-site/conductor.yml [hunk 1]
```diff
@@ -917,16 +917,6 @@ redirects:
     to: /docs/guides/protect-your-api/aspnet/next-steps/
   - from: /guides/protect-your-api/aspnet/require-authentication/index.html
     to: /docs/guides/protect-your-api/aspnet/require-authentication/
-  - from: /guides/protect-your-api/aspnetcore/before-you-begin/index.html
```
Comment (laura-rodriguez) [inline packages/@okta/vuepress-site/conductor.yml:920]: Should we map to aspnetcore3 instead?

**Review by laura-rodriguez — APPROVED** [review]
One question, then LGTM.

File: packages/@okta/vuepress-site/conductor.yml (hunk 1 above)
Comment (andriizhegurov-okta) [reply packages/@okta/vuepress-site/conductor.yml:920]: I'm not sure. I removed links to deleted docs, but did not touch any other, if links to aspnetcore3 existed they are still there

Comment (andriizhegurov-okta) [reply packages/@okta/vuepress-site/conductor.yml:920]: They might not be present in this redirection list though.

**Pull request merged**